lib/demo.rb

```diff
@@ -0,0 +1,63 @@
+###### This file is without connection Stack and Queue classes ############
+
+# require './Stack.rb'
+# require './Queue.rb'
+
+class JobSimulation
+  attr_reader :workers, :waiting, :roll, :fired, :hired
+
+  def initialize (jobs_available, job_seekers)
+    @waiting = Queue.new
+    @workers = Stack.new
+    @i = 1
+    # @workders = @waiting[0...jobs_available]
+    jobs_available.times do
+      @workers.push("Worker #{@i}")
+      @i += 1
+    end
```

Comment:
You don't need `i` here. You could have instead said:

```
jobs_available.times do |i|
  @workers.push("Worker #{i+1}")
end
```

Which would give you the same functionality but wouldn't create an unnecessary instance variable.